vision/getting-started/image_editing.ipynb

```diff
@@ -0,0 +1,573 @@
+{
```

Comment:
<p>Update the license year. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/489/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>